app/src/main/java/io/heckel/ntfy/msg/BroadcastService.kt

```diff
@@ -33,6 +33,8 @@ class BroadcastService(private val ctx: Context) {
         intent.putExtra("priority", notification.priority)
         intent.putExtra("muted", muted)
         intent.putExtra("muted_str", muted.toString())
+        intent.putExtra("attachment_name", notification.attachment?.name)
+        intent.putExtra("attachment_url", notification.attachment?.url)
```

Comment:
This will be null if it's not set. In the docs, you say "may be empty if not set" (https://github.com/binwiederhier/ntfy/pull/330). I'll change it to set it to an empty string if there is no attachment, or the value is null.

Reply:
:+1: